wsimod/nodes/storage.py

```diff
@@ -394,7 +449,6 @@ class River(Storage):
     # TODO non-day timestep
     def __init__(
         self,
-        depth=2,
```

Comment:
I'm not sure it's a great idea to remove arguments. As there will be many `config` files that specify `depth` for `River` nodes, and these will crash. Instead I would suggest to allow `depth` as an argument and have something like the following:

```
print('warning: the depth parameter is unused by River nodes because it is intended for capacity to be unbounded. It may be removed in a future version.')
```

Reply:
added - see commit 07e196be9930529d56612bf6894f25da6bfd8d75